src/common/src/lib.rs

```diff
@@ -43,6 +43,8 @@
 #![feature(negative_impls)]
 #![feature(bound_map)]
 #![feature(array_methods)]
+#![feature(register_tool)]
```

Comment:
https://doc.rust-lang.org/beta/unstable-book/language-features/register-tool.html doesn't seem well documented. Could you elaborate more on the usage here?

Reply:
Out of scope nevermind. It's introduced here: https://github.com/risingwavelabs/risingwave/pull/13763/files#diff-aef7438a31f20c6a6fc81e4f413e8f2231e2fb470823dc18110620ecc0da1843